python/pyspark/worker.py

```diff
@@ -254,7 +254,12 @@ def read_udfs(pickleSer, infile, eval_type):
         timezone = runner_conf.get("spark.sql.session.timeZone", None)
         safecheck = runner_conf.get("spark.sql.execution.pandas.arrowSafeTypeConversion",
                                     "false").lower() == 'true'
-        ser = ArrowStreamPandasSerializer(timezone, safecheck)
+        # NOTE: this is duplicated from wrap_grouped_map_pandas_udf
+        assign_cols_by_name = runner_conf.get(
+            "spark.sql.legacy.execution.pandas.groupedMap.assignColumnsByName", "true")\
```

Comment:
the `groupedMap` part of the conf doesn't really make sense here, but not sure if it's worth changing

Reply:
Based on above comment https://github.com/apache/spark/pull/23900/files#r260874304, if we are going to remove `spark.sql.legacy.execution.pandas.groupedMap.assignColumnsByName`, do we need to use this config here?

Reply:
I left it in to be consistent. I'd rather remove both of them in a separate PR in case there is some discussion about it.